arduino/smartcar/smartcar.ino

```diff
@@ -85,20 +85,33 @@ void loop() {
     const auto currentTime = millis();
     #ifdef __SMCE__
         static auto previousFrame = 0UL;
-        if (currentTime - previousFrame >= 65) {
+        if (currentTime - previousFrame >= oneSecond) {
```

Comment:
By doing this, your camera feed frame rate became 15 times slower. I don't see any good reason why it should be like that. Please revise.